src/proto/grpc/testing/messages.proto

```diff
@@ -135,7 +140,7 @@ message StreamingOutputCallRequest {
   Payload payload = 3;
 
   // Whether to request the server to compress the response.
-  bool request_compressed_response = 6;
+  bool response_compressed = 6;
```

Comment:
Also, because the value of "response_compressed" is shared for all the response messages, you can't make the "server_compressed_streaming" scenario alternate between compressed and uncompressed responses (which can be used as a verification that one can disable compression per-message on server side response writers).